spec/system/claims/view_organisations_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+feature "View organisations as a multi org user" do
+  around do |example|
+    Capybara.app_host = "https://#{ENV["CLAIMS_HOST"]}"
+    example.run
+    Capybara.app_host = nil
+  end
+
+  scenario "I sign in as persona Marry" do
+    persona = given_the_claims_persona("Mary")
+    and_persona_has_multiple_organisations(persona)
+    when_i_visit_claims_personas
+    when_i_click_sign_in(persona)
+    i_can_see_a_list_marys_claims_organisations(persona)
+  end
+end
```

Comment:
It would be good to also cover the scenario of users with only one organisation. Or is that already covered somewhere else?

```ruby
feature "View organisations " do
  scenario "when the user belongs to multiple organisations" do
    persona = given_the_claims_persona("Mary")
    and_persona_has_multiple_organisations(persona)
    when_i_visit_claims_personas
    when_i_click_sign_in(persona)
    i_can_see_a_list_marys_claims_organisations(persona)
  end

  scenario "when the user only has one organisation" do
    # ...
  end
end
```